src/coordination/coordinator_instance.cpp

```diff
@@ -354,6 +358,9 @@ void CoordinatorInstance::ForceResetCluster() {
   std::ranges::for_each(repl_instances_, [](auto &instance) { instance.StartFrequentCheck(); });
 
   MG_ASSERT(!raft_state_.IsLockOpened(), "After force reset we need to be in healthy state.");
+  auto leader_lock = std::unique_lock{leader_mutex_};
+  leader_set_up_.store(true);
+  become_leader_.notify_all();
```

Comment:
Why is this done in`ForceReset`? Also, what is the meaning of `leader_set_up_`, why it is saved in coordinator_instance?

Reply:
Doesn't `ForceReset` only change status of replication instances?

Reply:
Change as talked on call